apps/www/README.md

```
@@ -1,36 +1,55 @@
+   DATABASE_URL=postgresql://user:password@localhost:5432/mydb
+   
+   # Required: Upstash Redis configuration (https://upstash.com)
+   UPSTASH_REDIS_REST_TOKEN=your_upstash_redis_rest_token
+   UPSTASH_REDIS_REST_URL=your_upstash_redis_rest_url
+   ```
+
+2. **Install Prisma** and set up your database:
+   ```bash
+   pnpm install @prisma/client
+   npx prisma init
+  ```
```

Markdownlint (0.35.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)